lib/load-changelog-config.js

```diff
@@ -25,19 +25,13 @@ export default async ({ preset, config, parserOpts, writerOpts, presetConfig },
 
   if (preset) {
     const presetPackage = `conventional-changelog-${preset.toLowerCase()}`;
-    loadedConfig = importFrom.silent(__dirname, presetPackage) || importFrom(cwd, presetPackage);
+    loadedConfig = await (importFrom.silent(__dirname, presetPackage) || importFrom(cwd, presetPackage))(presetConfig);
   } else if (config) {
-    loadedConfig = importFrom.silent(__dirname, config) || importFrom(cwd, config);
+    loadedConfig = await (importFrom.silent(__dirname, config) || importFrom(cwd, config))();
```

Comment:
this should also accept the config, but we dont currently have a test for that case, so want to add those together

Reply:
in the interest of getting the beta promoted, i'm ok with completing this detail after this is merged